### PR TITLE
fix(SummaryStatistics): fix in sanity_filter

### PR DESCRIPTION
**Pull request by addramir**
## ✨ Context

This PR adds filtering by inf in summary statistics.

## 🚦 Before submitting

- [ ] Do these changes cover one single feature (one change at a time)?
- [ ] Did you read the [contributor guideline](https://opentargets.github.io/gentropy/development/contributing/#contributing-checklist)?
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you make sure there is no commented out code in this PR?
- [ ] Did you follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standards in PR title and commit messages?
- [ ] Did you make sure the branch is up-to-date with the `dev` branch?
- [ ] Did you write any new necessary tests?
- [ ] Did you make sure the changes pass local tests (`make test`)?
- [ ] Did you make sure the changes pass pre-commit rules (e.g `poetry run pre-commit run --all-files`)?

### CODE REVIEW

**Comment by addramir** [comment]
@project-defiant could you please have a look on it? for some reason, it dosen't work in my hands and fails "make test"

**Comment by project-defiant** [comment]
@addramir it should be ready to run

**Comment by addramir** [comment]
@project-defiant I made small changes in sanity filter function and test. Please have a look. If you agree - you can approve it.

File: tests/gentropy/method/test_qc_of_sumstats.py [hunk 1]
```diff
@@ -88,19 +89,19 @@ def test_sanity_filter_remove_inf_values() -> None:
...
-            530.4275,
-            289,
+            5.4275,
+            -2890,
             None,
             0.4671,
         ),
     ]
-    input_df = spark.createDataFrame(data=data, schema=SummaryStatistics.get_schema())
-    input_df.printSchema()
```
Comment (project-defiant) [inline tests/gentropy/method/test_qc_of_sumstats.py]: I missed this one, good catch

**Review by project-defiant — APPROVED** [review]

**Comment by project-defiant** [comment]
@addramir all good for merge

**Pull request merged**